src/coreclr/tools/aot/ILCompiler.Compiler/IL/ILImporter.Scanner.cs

```diff
@@ -1069,17 +1069,39 @@ private void ImportBinaryOperation(ILOpcode opcode)
             {
                 case ILOpcode.add_ovf:
                 case ILOpcode.add_ovf_un:
-                case ILOpcode.mul_ovf:
-                case ILOpcode.mul_ovf_un:
                 case ILOpcode.sub_ovf:
                 case ILOpcode.sub_ovf_un:
                     _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.Overflow), "_ovf");
                     break;
+                case ILOpcode.mul_ovf:
+                case ILOpcode.mul_ovf_un:
+                    // Required for ARM
```

Comment:
```suggestion
```
This comment is redundant with the code. The code describes the intent well enough.